### ISSUE TITLE
Tests cannot be run

### ISSUE THREAD

**Issue by axelniklasson** [i-like-robots/react-tags#131]
There seems to be a problem with jsdom when running the tests locally, as well on travis as can be seen [here](https://travis-ci.org/i-like-robots/react-tags/jobs/412544078). It should be noted that they can be run when using the second latest version `11.11.0` instead of the most recent version `11.12.0`.

Will look into this and see if I can find a fix. If so, it will be added to my current [pull request](https://github.com/i-like-robots/react-tags/pull/130).

````
➜  react-tags git:(master) ✗ yarn test
yarn run v1.7.0
$ npm run lint && npm run build:es5

> react-tag-autocomplete@5.5.1 lint /Users/axni/dev/react-tags
> standard lib/*.js spec/*.js


> react-tag-autocomplete@5.5.1 build:es5 /Users/axni/dev/react-tags
> buble lib --objectAssign -o dist-es5

$ jasmine

/Users/axni/dev/react-tags/node_modules/jsdom/lib/jsdom/browser/Window.js:257
        throw new DOMException("localStorage is not available for opaque origins", "SecurityError");
        ^
SecurityError: localStorage is not available for opaque origins
    at Window.get localStorage [as localStorage] (/Users/axni/dev/react-tags/node_modules/jsdom/lib/jsdom/browser/Window.js:257:15)
    at Object.<anonymous> (/Users/axni/dev/react-tags/spec/helpers/dom.js:23:23)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at /Users/axni/dev/react-tags/node_modules/jasmine/lib/jasmine.js:99:5
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
````

**Comment by axelniklasson** [comment]
Fixed in 81e746a by supplying URL when instantiating jsdom in `spec/helpers/dom.js`. Tests can now be run.

**Comment by i-like-robots** [comment]
Thank you. We've been seeing this error for the past week or so at work and I've added the fix to so many codebases!
